From 08f0195e527c024b803a15b1715ee38a8b27d2f2 Mon Sep 17 00:00:00 2001
From: Tobias Schaffner <tobias.schaffner@siemens.com>
Date: Mon, 13 Nov 2023 15:46:39 +0100
Subject: [PATCH] startfive-visionfive2: Allow kernel commandline tuning

The bitbake variable IMAGE_KERNEL_CMDLINE_TUNE can be used to pass
additional kernel commandline arguments.

Signed-off-by: Tobias Schaffner <tobias.schaffner@siemens.com>
---
 .../scripts/lib/wic/canned-wks/starfive-visionfive2.wks.in      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta-isar/scripts/lib/wic/canned-wks/starfive-visionfive2.wks.in b/meta-isar/scripts/lib/wic/canned-wks/starfive-visionfive2.wks.in
index 0f3d76a9..c127325b 100644
--- a/meta-isar/scripts/lib/wic/canned-wks/starfive-visionfive2.wks.in
+++ b/meta-isar/scripts/lib/wic/canned-wks/starfive-visionfive2.wks.in
@@ -20,4 +20,4 @@ part /boot --source bootimg-efi-isar --sourceparams "loader=systemd-boot,dtb=${STARFIVE_VISIONFIVE2_DTB}" --use-
 # rootfs
 part / --source rootfs --use-uuid --fstype ext4 --mkfs-extraopts "-T default" --label image --active --align 1024 --exclude-path=boot/
 
-bootloader --ptable gpt --timeout=0 --append "console=tty0 console=ttyS0,115200 earlycon rootwait"
+bootloader --ptable gpt --timeout=0 --append "console=tty0 console=ttyS0,115200 earlycon rootwait ${IMAGE_KERNEL_CMDLINE_TUNE}"
-- 
2.34.1

